security/advancedtls/advancedtls.go

```diff
@@ -119,8 +124,9 @@ type GetRootCAsResults = RootCertificates
 
 // RootCertificateOptions contains options to obtain root trust certificates
 // for both the client and the server.
-// At most one option could be set. If none of them are set, we
-// use the system default trust certificates.
+// At most one option should be set. If none of them are set, we use the system
```

Comment:
What is an "option"?  Is each option a single field in this struct?  If so, maybe "at most one field should be set"?

Reply:
Changed, good call